Seneca-CDOT/starchart
# Resolves #7 by initializing Remix's Blues Stack

## Description
This PR resolves #7 by providing an initial Remix's Blues Stack for the Starchart project. 

### Pre-requisites to Testing
- Install and set up Postgres so the Remix app runs successfully. I used [Postgres App Client](https://postgresapp.com/downloads.html) as it does the job quickly and is easy to use
### Steps to Test
1. Install dependencies using `npm install`
2. Run `npx remix init`
3. Run `npm run docker`
4. Run `npm run setup`
5. Run `npm run build`
6. Run `npm run dev`

Now, navigate to `localhost:3000` to see Remix App up and running:

<img width="1555" alt="Screenshot 2023-01-26 at 10 27 52 PM" src="https://user-images.githubusercontent.com/50856799/214912300-f182de4c-3560-4d40-9b57-09c3044f9955.png">

### Additional Information
- To avoid duplicate files, Remix's Blues Stack's code from `docker-compose.yml` and `.gitingore`, was added to existing Starchart files with the same name
- `README.md` file from Remix's Blues Stack is titled `README-remix.md`, it can be moved to the wiki or somewhere else

### Resource
https://github.com/remix-run/blues-stack
